mantidimaging/gui/windows/image_load_dialog/test/presenter_test.py

```diff
@@ -23,13 +23,23 @@ def setUp(self):
         self.p = LoadPresenter(self.v)
 
     def test_do_update_sample_with_no_selected_file(self):
-        self.v.select_file.return_value = None
+        self.p.do_update_sample(None)
```

Comment:
NITPICK: though it is bordering being outside the scope of the work done within this PR, I feel that `self.v` and `self.p` are not particularly descriptive names. How would you feel about updating them as part of your changes or would you prefer we do this as a separate flash review if you agree?

Reply:
Its a shorthand we use in lots of tests, so would be a big change if we did it. Maybe a cleanup to consider in the future.